src/outputs.c

```diff
@@ -225,6 +225,19 @@ void deinit_outputs() {
         evdev = NULL;
     }
 }
+void singletap(uint32_t timestamp_ms, imu_quat_type quat, imu_euler_type velocities, bool ipc_enabled,
+                       bool imu_calibrated, ipc_values_type *ipc_values) {
+    libevdev_uinput_write_event(uinput, EV_KEY, BTN_LEFT,1);
```

Comment:
We should be doing something like the checks we have below, since a double-click only makes sense in these contexts (VR-Lite mouse mode specifically, should probably ignore for joystick mode):
```
    if (uinput) {
        if (is_joystick_mode(context.config)) {
            ...
        } else if (is_mouse_mode(context.config)) {
```